products/linuxmint.md

```diff
@@ -99,7 +99,6 @@ Linux Mint releases follows [the support cycle](https://linuxmint.com/download_a
 ## Ubuntu Package Support
 
 * Linux Mint has its own repositories for Linux Mint specific packages, but relies directly on Ubuntu repositories for the majority of the package base, including its mirrors. 
```

Comment:
I think the distinction for packages installed from `universe` is still important enough to merit a mention:

```suggestion
* Linux Mint has its own repositories for Linux Mint specific packages, but relies directly on Ubuntu repositories for the majority of the package base, including its mirrors. 
* Packages installed from the `universe` repository are maintained by the community and [they may not receive timely security updates](https://help.ubuntu.com/community/Repositories#Universe).
```

We should add the same to the ubuntu page.